### ISSUE TITLE
Inconsistent styles in modal headers

### ISSUE THREAD

**Issue by viddo** [CartoDB/cartodb#4175]
I noticed that we are actually not consistent in how the modals are rendered, which is due to having duplicated CSS here and there. See these screenshots:
![screen shot 2015-06-23 at 18 54 38](https://cloud.githubusercontent.com/assets/978461/8311865/913ae3da-19d9-11e5-8eae-6c8c189fcc5e.png)
![screen shot 2015-06-23 at 18 54 45](https://cloud.githubusercontent.com/assets/978461/8311866/913e5006-19d9-11e5-82d4-1ff93f55e225.png)
![screen shot 2015-06-23 at 18 54 53](https://cloud.githubusercontent.com/assets/978461/8311867/9140983e-19d9-11e5-8dbb-b9f54e7c335c.png)
![screen shot 2015-06-23 at 18 55 17](https://cloud.githubusercontent.com/assets/978461/8311868/914495b0-19d9-11e5-9331-09dcfb7ff03f.png)

Basically the items in the header have different sizes (i.e. the icons, title and subtitles), as well as not being consistent when it comes to the color (different shades). 

Not sure if the issue was due to not being consistent in the sketch files, or a "translation" issue when we implemented them, but either way I think we would benefit from consolidating them to K.I.S.S.

Opinions? @saleiva @josecruz @javierarce @xavijam

**Comment by saleiva** [comment]
totally.
@josecruz can you take a look at the sketch files and see how many different do we have? 
We can create another sketch file to work on this, and once we unify and create the rules, implement on all sketch files using this new modals

**Comment by viddo** [comment]
(summary from email conversation)

The goal of this is to remove unnecessary css and to gain in consistency. Based on the analysis we will end having 4 different variations for the icons on the harder of the modals.
- Dark grey (#666) for actions that are neutral.
- Orange for actions that are ‘dangerous’ or for actions that might have unnexpected consequences, such as renaming a table.
- Red for critical actions such as removing things.
- Light grey (#CCC) for the icons on multi-step headers that are reflecting the non-active state.

We shouldn’t have any icon on the header of any modal that is not any of those colors. Not blue, not other greys. 

Variations are located in _unified-headers-0.1.sketch_

**Comment by viddo** [comment]
Note from while working on [georeferences fixes](https://github.com/CartoDB/cartodb/issues/4262), we have duplicated styles for the icons (`.LayoutIcon` and `Dialog-headerIcon`), we should be able to reuse the `.LayoutIcon` and just `.Dialog-headerIcon` class for the positioning and margin in the context of modal header.
